scaleout/studio/values.yaml

```diff
@@ -56,3 +56,12 @@ ingress:
     - secretName: platform-tls-secret
       hosts:
         - platform.demo.scaleout.se
+
+rabbit:
+  username: admin
+  password: sp3ctr4l
```

Comment:
can you set this to a generated passwordlike string? like 69QMT5yXtEwo7BSCJsemMWnH7Bx7RUNDSL724IRrAvU=